src/pages/[platform]/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -114,24 +114,16 @@ Amplify.API.mutate(ModelMutation.create(team),
 <InlineFilter filters={["swift"]}>
 
 ```swift
-Task {
-    do {
-        let team = Team(mantra: "Go Frontend!")
-        let createTeamResult = try await Amplify.API.mutate(request: .create(team))
-        guard case .success = createTeamResult else {
-            print("API response: \(createTeamResult)")
-            return
-        }
-
-        let member = Member(name: "Tim", team: try createTeamResult.get()) // Directly pass in the post instance
-        let createMemberResult = try await Amplify.API.mutate(request: .create(member))
-        guard case .success = createMemberResult else {
-            print("API response: \(createMemberResult)")
-            return
-        }
-    } catch {
-        print("Create team or member failed", error)
-    }
+do {
+    let team = Team(mantra: "Go Frontend!")
+    let createdTeam = try await Amplify.API.mutate(request: .create(team)).get()
+
+    let member = Member(
+        name: "Tim",
+        team: createdTeam) // Directly pass in the post instance
+    let createdMember = try await Amplify.API.mutate(request: .create(member))
+} catch {
+    print("Create team or member failed", error)
 }
```

Comment:
<img width="1006" alt="image" src="https://github.com/aws-amplify/docs/assets/1365977/4688a3ad-9b75-4b7b-94f1-0ed61c97b899">

Most of these changes remove the Task since that seems unnecessary. Developers can choose to put this in an async function or wrap it in a Task based on their preference. 

The guard statements have been removed since the failure case will be caught in the catch block.